distributed/client.py

```diff
@@ -343,7 +343,7 @@ class Client(object):
     distributed.scheduler.Scheduler: Internal scheduler
     """
     def __init__(self, address=None, start=True, loop=None, timeout=3,
-                 set_as_default=True, scheduler_file=None):
+                 set_as_default=True, scheduler_file=None, connection_kwargs=None):
```

Comment:
Depending on the config.yaml file is probably a bad choice for anything except debugging options.  It is annoying to guarantee this presence of this file in all deployment scenarios.  Job schedulers often make it far easier to pass keywords to scripts than to populate environments with files. 

However, it would be nice not to have to specify `--certfile` etc.  whenever starting up dask-workers.  I can see how this would be tedious.  I wonder if there is some way to initially seed connection_kwargs with a section in the config file.